File: Module 2/27_Fibonacci_Factorial/C/Factorial.c
#include <stdio.h>

int factorial(int n) {
    if (n==0)
        return 1;
    else
        return (n * factorial(n-1));
}

void main() {
    int n = 0;
    printf("Please Enter the Number to see its Factorial : ");
    scanf("%d", &n);
    printf("Factorial of %d : %d", n, factorial(n));
}

File: Module 2/27_Fibonacci_Factorial/Java/Factorial.java
import java.util.Scanner;

public class Factorial {
    public static int factorial(int n) {
        if (n == 0)
            return 1;
        else
            return (n * factorial(n - 1));
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Please Enter the Number to see its Factorial: ");
        int n = sc.nextInt();
        System.out.println("Factorial of " + n + " : " + factorial(n));
    }
}
